core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -348,9 +348,9 @@ $(document).ready(function () {
 
             // prepare data for executor summary table
             stageExecutorSummaryInfoKeys = Object.keys(responseBody.executorSummary);
+            var executorDetailsMap = {};
```

Comment:
@gengliangwang . Is this filled correctly always? I'm hitting the following error during reviewing this patch.

![screenshot](https://user-images.githubusercontent.com/9700541/50068441-c12b7780-017a-11e9-87ee-8c31d8e587f4.png)

Reply:
I think it is caused by the cache of browser. The javascript might be using the cached one.

Reply:
Try loading the page without cache should work: https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache

Reply:
@gengliangwang you're sure it works locally? I just want to make sure as I'm not sure how well tests cover this case